sample_data/pomodoro.json

```diff
@@ -0,0 +1,5 @@
+{
+    "defaultTime": "24.0",
```

Comment:
This format is just what I need for what any storage the Pomodoro function needs. Great